celements-component/src/main/java/org/xwiki/component/annotation/Component.java

```diff
@@ -39,24 +40,27 @@
 @Retention(RUNTIME)
 @Target(TYPE)
 @Inherited
-public @interface Component
-{
-    /**
-     * The hint value. Can be any text value. It's used to differentiate one component implementation
-     * from another one.
-     */
-    String value() default "";
+public @interface Component {
 
-    /**
-     * Sometimes you want to register the component several times with different hints. In this case the
-     * default annotation value is ignored and the passed list of hints is used instead.
-     */
-    String[] hints() default { };
-    
-    /**
-     * When specified forces the component to be registered as many times as there are roles specified. Otherwise
-     * the superclass/interface hierarchy is scanned for ComponentRole annotations and the component is registered
-     * under all roles found. 
-     */
-    Class< ? >[] roles() default { };
+  /**
+   * The hint value. Can be any text value. It's used to differentiate one component implementation
+   * from another one.
+   */
+  String value() default "";
+
+  /**
+   * Sometimes you want to register the component several times with different hints. In this case
+   * the
+   * default annotation value is ignored and the passed list of hints is used instead.
```

Comment:
```suggestion
   * the default annotation value is ignored and the passed list of hints is used instead.
```
Please check double line-breaks for all comments